infrastructure/cluster/flux-v2/jhub/jhub-release.yaml

```diff
@@ -162,10 +168,20 @@ spec:
           - name: eulake-escape-tmpdata # mounts the EOS RSE needed for the Rucio JupiterLab extension
             mountPath: /eos/scratch-space
             mountPropagation: HostToContainer
+          - name: eulake-escape-data-test-fusemount
+            mountPath: /eos/test-fusemount
+            mountPropagation: HostToContainer
+            readOnly: true
+          - name: eulake-escape-tmpdata-test-fusemount
+            mountPath: /eos/test-fusemount
+            mountPropagation: HostToContainer
       image:
         name: ghcr.io/vre-hub/vre-singleuser
         tag: sha-40199b2
         pullPolicy: Always
+      nodeSelector:
+        # makes sure EOS eulake fuse mount is done on nodes designated to have the fuse mount
+        jupyter-role: singleuser
```

Comment:
Nodes need to be tagged in advance for this to work. There may be a cool way to do this with Flux or tf to keep it as code.